src/transformers/data/processors/squad.py

```diff
@@ -141,11 +144,21 @@ def squad_convert_example_to_features(example, max_seq_length, doc_stride, max_q
     span_doc_tokens = all_doc_tokens
     while len(spans) * doc_stride < len(all_doc_tokens):
 
+        # Define the side we want to truncate / pad and the text/pair sorting
+        if tokenizer.padding_side == "right":
```

Comment:
Can you add a comment about which contexts padding_side='right' should be used? I am super confused by this code.